### ISSUE TITLE
kl_weight

### ISSUE THREAD

**Issue by YangNuoCheng** [LasSino/fedbnn_code#1]
The 'kl_weight' in configs is '1'.
However, the value of KL-loss is about 1e-8 while the  CrossEntropy-Loss based on dataset is about 1e-1.
How can we maintaining a balance between new knowledge (as shown in CrossEntropy-Loss) and old knowledge (as shown in KL-loss) by combing them with kl_weight=1?

**Comment by LasSino** [comment]
Sorry for the late reply. I am working towards other businesses and is not actively focusing on this repository.
You are right. The values provided in the config file are just for reference, mainly used to demonstrate available options. As stated in the paper, KL Weight is actually considered as a hyper-parameter. Therefore, it is recommended to tune it through pre-training. In my personal experience, the example value of kl_weight (1) is not likely to be an ideal value for most training processes.
